test/__snapshots__/variants.js.snap

```diff
@@ -39,6 +39,8 @@ exports[`variants > negative 1`] = `
 exports[`variants > pseudo 1`] = `
 "/* layer: default */
 .focus\\:border:focus{border-width:1px;}
+.aria-\\[invalid\\=spelling\\]\\:s-bg-primary[aria-invalid=spelling],
+.data-\\[is-active\\]\\:s-bg-primary[data-is-active],
```

Comment:
Hmm, are these test really testing the right thing? 🤔

Reply:
Ah, it is because I used the same color as the other selector, that why there is a `,` at the end. 😅